src/__screenshot_tests__/input-fields-screenshot-test.tsx

```diff
@@ -521,6 +521,28 @@ test.each`
     expect(await fieldWrapper.screenshot()).toMatchImageSnapshot();
 });
 
+test.only.each`
```

Comment:
I think you forgot to remove `only` from here

Reply:
ouch!